dockerbot.py

```diff
@@ -1,52 +1,260 @@
-            bot.sendMessage(chat_id, "Signed")
+            Image = '/opt/dockerbot/images/webtop_approval.png'
+            import Webtop_Health_Statements
+            if Webtop_Health_Statements.sign(v_UserId, v_UserKey, Image) == 1:
+                time.sleep(2)
+                bot.sendPhoto(chat_id=chat_id, photo=open(str(Image), 'rb'))
+                logger.info(f"[{message_id}] Return result to command {command}. Result image path: {Image}")
+                bot.sendMessage(chat_id, "Signed")
+            else:
+                bot.sendMessage(chat_id, "Well, Somthing went wrong, please check the logs for more info")
         except Exception as ex:
             logger.exception(f"[{message_id}] Failed to handle command. Msg: {command}")
             bot.sendMessage(chat_id, f"ERROR: {str(ex)}")
 
+
+    if command == '/sign_mashov':
+         try:
+             if v_MASHOV_NUMBER_OF_KIDS >= 1 and list['mashov']['kid1']['MASHOV_USER_ID_KID'] != None:
+                 for Mashov_Kid_Number in range(1, v_MASHOV_NUMBER_OF_KIDS, 1):
+                     bot.sendMessage(chat_id,"Starting Sign process at https://web.mashov.info/students/login for Kid Number: " + str(Mashov_Kid_Number))
+                     Prep_Switch_MASHOV_USER_DICT_ID_KID = list['mashov']['kid'+str(Mashov_Kid_Number)]['MASHOV_USER_ID_KID']
+                     Prep_Switch_MASHOV_USER_DICT_ID_PWD = list['mashov']['kid'+str(Mashov_Kid_Number)]['MASHOV_USER_PWD_KID']
+                     Prep_Switch_MASHOV_USER_DICT_ID_SCHOOL_ID = list['mashov']['kid'+str(Mashov_Kid_Number)]['MASHOV_SCHOOL_ID_KID']                     
+                     subprocess.check_output(['python', '/opt/dockerbot/Mashov_Health_Statements.py', '-u', Prep_Switch_MASHOV_USER_DICT_ID_KID, '-p', Prep_Switch_MASHOV_USER_DICT_ID_PWD , '-s', Prep_Switch_MASHOV_USER_DICT_ID_SCHOOL_ID, '-kn', str(Mashov_Kid_Number)])
+             else:
+                 bot.sendMessage(chat_id, "mashov NOT configured")
+                 config_mashov = 0
+
+             for file in os.listdir("/opt"):
+                 if file.endswith(".png") and file.startswith("mashovkid"):
+                     Image = os.path.join("/opt", file)
+                     bot.sendPhoto(chat_id=chat_id, photo=open(str(Image), 'rb'))
+                     os.remove(str(Image))
+                     logger.info(f"[{message_id}] Return result to command {command}. Result image path: {Image}")
+             if config_mashov != 0:
+                 bot.sendMessage(chat_id, "Signed")
+         except Exception as ex:
+             logger.exception(f"[{message_id}] Failed to handle command. Msg: {command}")
+             bot.sendMessage(chat_id, f"ERROR: {str(ex)}")
+
     msg = f"Done message handling: {command}"
     logger.info(f"[{message_id}] {msg}")
 
 
+    if command == '/sign_all':
+        if list['auto_sign']['edu'] == 1:
+            Image = '/opt/dockerbot/images/edu_approval.png'
```

Comment:
the Image is not in the Repo

Reply:
There is an image foreach healthstatment.
if the sign sucsseded the image than will be created and send